sandbox/src/pages/Buttons.tsx

```diff
@@ -238,5 +239,17 @@ export const Buttons = () => (
         <p>{`<ButtonHelp backgroundSecondary />`}</p>
       </code>
     </CodeDrawer>
+
+    <h4>Button Tab</h4>
+    <div className="row">
+      <ButtonTab title={"TabA"} />
+      <ButtonTab active title={"TabB"} />
```

Comment:
Here you should showcase also the "onClick" and the state changing, since there are actions and it is not a simple Button component.
Meaning that in the showcase, there should be actions happening when the user is clicking things around

Reply:
I will make note on this for the new PR